src/ContainerBuilder.php

```diff
@@ -55,16 +55,6 @@ public function build()
                     $c['jira_client']
                 );
             },
-            'listener.pull_request.test' => function ($c) {
-                return new Listener\TestPullRequest(
-                    $c['bamboo_username'],
```

Comment:
I think you can then also remove the parameters from:
https://github.com/phpbb/dev-hooks/blob/79ddbe7ecbd3bf2eb6471ebca3ff7e7608aeeb1c/config/parameters.php.sample#L7

Reply:
Done